### ISSUE TITLE
Find All References doesn't work in 15.8.0 preview 5

### ISSUE THREAD

**Issue by isaacabraham** [Microsoft/visualfsharp#5479]
Using Visual Studio 15.8.0 preview 5, trying to find all references for even trivial code shows no results:

```fsharp
let x =
    let z = 5 // Shift + F12 on 'z'
    z + 5
```
This correctly brings up the Find All References dialog, but says "Search found no results". However, moving the caret over `z` correctly highlights all references inside the editor pane.

Copy Info from About shown here:

```
Microsoft Visual Studio Enterprise 2017 Preview
Version 15.8.0 Preview 5.0
VisualStudio.15.Preview/15.8.0-pre.5.0+27924.0
Microsoft .NET Framework
Version 4.7.03056

Installed Version: Enterprise

C# Tools   2.9.0-beta8-63119-10
C# components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Common Azure Tools   1.10
Provides common services for use by Azure Mobile Services and Microsoft Azure Tools.

NuGet Package Manager   4.6.0
NuGet Package Manager in Visual Studio. For more information about NuGet, visit http://docs.nuget.org/.

ProjectServicesPackage Extension   1.0
ProjectServicesPackage Visual Studio Extension Detailed Info

ResourcePackage Extension   1.0
ResourcePackage Visual Studio Extension Detailed Info

ResourcePackage Extension   1.0
ResourcePackage Visual Studio Extension Detailed Info

Visual Basic Tools   2.9.0-beta8-63119-10
Visual Basic components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Visual F# Tools 10.2 for F# 4.5   15.8.0.0.  Commit Hash: 816a5a25b4a8bea43b72eb41bb55589d7605a1cd.
Microsoft Visual F# Tools 10.2 for F# 4.5

Visual Studio Code Debug Adapter Host Package   1.0
Interop layer for hosting Visual Studio Code debug adapters in Visual Studio
```

**Comment by cartermp** [comment]
Note - rename also does not find symbols but semantic highlighting does, so this is likely related to finding documents in a solution.

We'll investigate; I used FAR on an internal build yesterday (for 15.8 RTW) so this _may_ be fixed, but we'll have to double check. @TIHan made an upstream fix to Roslyn in this space that is not available in Preview 5.

**Comment by cartermp** [comment]
Confirmed this failing on 28007.0.d15.8 (internal), so I must have been using FAR with a 15.7 build.

@Pilchie for FYI

**Comment by TIHan** [comment]
@isaacabraham thank you so much for testing this.

I am testing the latest dev15.8 on visualfsharp and dev15.8.x on Roslyn together in the same hive. I'm not able to reproduce the issue, but I can reproduce the issue in preview 5.

The problem was most likely occurring in Roslyn and has since been fixed. We will be testing another build today.

**Comment by cartermp** [comment]
FYI we have verified that this is an issue with 15.8, and in doing so have uncovered another issue in our dependencies that may be either affecting or masking the underlying cause. First, we'll need to update our Roslyn binaries, deal with that change, and then get this addressed.

**Comment by TIHan** [comment]
We figured out the cause. It was because the glyph tags API moved to another place in the shipped Roslyn dev15.8. F# didn't update to that same package of where it moved and caused a crash. Updating the package resolves this issue.

**Comment by cartermp** [comment]
Fixed by #5485 

Closing, and will report back on _where_ this makes it into.

**Comment by cartermp** [comment]
This will be in 15.8 and the 15.9 preview.
